README.md

```diff
@@ -1,2 +1,5 @@
 # coverage-crawler
+
+[![Build Status](https://travis-ci.org/marco-c/coverage-crawler.svg?branch=master)](https://travis-ci.org/marco-c/coverage-crawler)
```

Comment:
This should be part of a separate PR, we should try to make each PR self-contained

Reply:
Should I change this now or prevent a mess in the next time?

Reply:
Change this now and move it to a separate PR

Reply:
@marco-c I am sorry for such messy PR. I've changed the code to use just urlretrieve as you suggested, the taskId is used for unique identification of coverage builds as tar.bz2 archive.